backend/users-service/app/prisma/schema.prisma

```diff
@@ -0,0 +1,31 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mongodb"
+  url      = env("DATABASE_URL")
+}
+
+model Question {
+  id          String   @id @default(auto()) @map("_id") @db.ObjectId
+  category    String
+  description String
+  difficulty  String
+  id_         String   @map("id")
```

Comment:
btw what does `id_` do?

Reply:
> nit: btw what does `id_` do?

Actually I copy pasted from @Josephaedan's question schema, @Josephaedan do you know what it represents? Probably can remove it for this custom schema